assets/src/blocks/amp-story/index.js

```diff
@@ -122,11 +122,7 @@ export const settings = {
 					)
 				}
 				<amp-story-grid-layer template="fill" style={ overlayStyle }>
```

Comment:
This looks like it doesn't have a closing tag. Probably something that happened with merging `amp-stories-redux`.

Reply:
Whoops, indeed a merge conflict resolving gone wrong